deployments/apiv2/services/users/src/main.py

```diff
@@ -78,142 +78,121 @@ async def startup():
-        if isinstance(details.customer_id, uuid.UUID):
-            # if a UUID was given in the request then check against the customer ID
-            select_query = (
-                "SELECT password, id, failed_login_attempts, suspended, customer_id FROM users "
-                "WHERE deleted_at IS NULL AND name=$1 AND customer_id=$2 AND verified='t'"
+    try:
+        async with request.state.pgpool.acquire() as con:
+            select_query_result = await con.fetchrow(
+                "SELECT password, id, failed_login_attempts, suspended "
+                "FROM customers WHERE deleted_at IS NULL AND email=$1",
+                email,
             )
-        else:
-            # if no UUID given, the check against the customer account alias
-            # TODO should make sure an alias is actually set here?
-            select_query = (
-                "SELECT u.password, u.id, u.failed_login_attempts, u.suspended, u.customer_id "
-                "FROM users AS u JOIN customers AS c ON u.customer_id=c.id "
-                "WHERE u.deleted_at IS NULL AND u.name=$1 AND c.alias=$2 AND u.verified='t'"
+            pw = details.password.get_secret_value()
+            # verify password, else raise LoginError
+            await _verify_password(con, account_type, pw, select_query_result)
+            # get scopes from account_scopes table
+            scope = await _get_account_scope(con, select_query_result["id"], True)
+            # get list of scopes that a customer can assign to it's users
+            avail_user_scopes = _get_user_scopes_from_customer(scope)
+            # TODO decide how to show customer accounts usage data for multiple products, defaulting to mantarray now
+            # check usage for customer account
+            usage_quota = await check_customer_quota(con, str(select_query_result["id"]), "mantarray")
+
+            # if login was successful, then update last_login column value to now
+            await con.execute(
+                "UPDATE customers SET last_login=$1, failed_login_attempts=0 WHERE deleted_at IS NULL AND email=$2",
+                datetime.now(),
+                email,
             )
-        select_query_params = (username, str(details.customer_id))
 
-        update_last_login_query = (
-            "UPDATE users SET last_login=$1, failed_login_attempts=0 "
-            "WHERE deleted_at IS NULL AND name=$2 AND customer_id=$3 AND verified='t'"
+            tokens = await _create_new_tokens(con, select_query_result["id"], None, scope, account_type)
+            return LoginResponse(tokens=tokens, usage_quota=usage_quota, user_scopes=avail_user_scopes)
+
+    except LoginError as e:
+        raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail=str(e))
+    except Exception:
+        logger.exception("POST /login/customer: Unexpected error")
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@app.post("/login", response_model=LoginResponse)
```

Comment:
Did we decide not to also add `/login/user`? We could just do 
```
@app.post("/login/user", response_model=LoginResponse)
async def _login_user(request: Request, details: UserLogin):
   return await loging_user(request, details)
```
Or something like that so they both work